dictionaries/en.dictionary.itop.core.php

```diff
@@ -29,6 +29,8 @@
 	'Core:UnknownObjectTip' => 'The object could not be found. It may have been deleted some time ago and the log has been purged since.',
 
 	'Core:UniquenessDefaultError' => 'Uniqueness rule \'%1$s\' in error',
+	'Core:CheckConsistencyError' => 'Consistency rules not followed: %1$s',
+	'Core:CheckValueError' => 'Unexpected value for attribute \'%1$s\': %3$s',
```

Comment:
This changes the printed string, it's now displaying the attribute label instead of the attribute code.
Proposed change so it displays everything:
```suggestion
	'Core:CheckValueError' => 'Unexpected value for attribute \'%1$s\' (%2$s) : %3$s',
```